Prolog/models/gameState.pl

```diff
@@ -16,18 +17,22 @@
 score(GameState, Score) :-
   GameState=gameState(bird(_), pipeGroups(_), score(Score), highestScore(_), screenType(_)).
 
-incrementScore(GameState, Score, Increment) :-
-  NewScore is Score+Increment,
-  GameState=gameState(bird(_), pipeGroups(_), score(NewScore), highestScore(_), screenType(_)).
-
 highestScore(GameState, HighestScore) :-
   GameState=gameState(bird(_), pipeGroups(_), score(_), highestScore(HighestScore), screenType(_)).
 
 screenType(GameState, ScreenType) :-
-  isScreenType(ScreenType),
   GameState=gameState(bird(_), pipeGroups(_), score(_), highestScore(_), screenType(ScreenType)).
 
 isScreenType(ScreenType) :-
   playingScreenType(ScreenType);
   pausedScreenType(ScreenType);
   gameOverScreenType(ScreenType).
+
+incrementScore(GameState, Increment) :-
```

Comment:
We need to create a new game state to increment the score, so this method should have a return variable:
```prolog
incrementScore(GameState, Increment, NewGameState):-
...
```